src/objects.js

```diff
@@ -1,39 +1,72 @@
 export const createPerson = (name, age) => {
-  // your code here
+  // creates an object with the given name and age properties
+  let person = {
+    name: name,
+    age: age
+  };
+
+  return (person);
+
 };
 
 export const getName = (object) => {
-  // your code here
+  // returns the name property of the object
+  return(object["name"]);
+
```

Comment:
There shouldn't ever be empty lines directly after a code block (`{...}`), or right before a code block ends

Reply:
Also ensure consistency with the quotes you use (either single or double, never both). Most JS developers prefer to use single quotes only.